### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by ace0625**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+	},
+	"devDependencies": {
+		"babel-jest": "18.0.0",
+		"babel-preset-react-native": "1.9.1",
+		"jest": "18.1.0",
+		"react-test-renderer": "15.4.1"
+	},
+	"jest": {
+		"preset": "react-native"
+	}
```
Comment (ochanje210) [inline package.json]: indentation seems bit too long?

Comment (ace0625) [reply package.json]: Sorry. Fixed.

**Comment by ochanje210** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-client/pull-request/4/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-client)

**Comment by ochanje210** [comment]
shipit

**Pull request merged**